html/semantics/forms/the-fieldset-element/accessibility/legend-role-group-manual.html

```diff
@@ -0,0 +1,7 @@
+<!doctype html>
+<title>fieldset accessibility test: legend role=group aria-labelledby=fieldset</title>
+<fieldset id=fieldset>
+ <legend role=group aria-labelledby=fieldset>Foo</legend>
```

Comment:
a tricky one, the output looks reasonable, but I didn't run it through the spec name computation alg

Reply:
Yeah the spec side is pretty messy. I filed an issue but can't find it now. Basically I think accname should have hooks for html-aam to use when defining fieldset/legend.